package.json

```diff
@@ -31,6 +31,7 @@
     "commit": "git-cz",
     "generate-metadata": "node ./scripts/generate_metadata/index.mjs",
     "type-check": "tsc --noEmit",
+    "type-check:cypress": "tsc -p ./cypress/tsconfig.json",
```

Comment:
I think the additional time it would take to type check the cypress directory is negligible enough to mean we can just go with the following:

`"type-check": "tsc --noEmit && tsc -p ./cypress/tsconfig.json",`

`"precompile": "npm run type-check && npm run clean-lib && npm run build && npm run build:types && npm run copy-files",`

Reply:
We will need to amend the `type-check:watch` script if we do this, since with this change, the compiler will only check for `cypress/` changes in watch mode, not `src/` changes as well.

Reply:
I've updated the `type-check:watch` to use the `concurrently` package to run the compiler on the `src/` and `cypress/` directories in watch mode in parallel.